private/buf/bufgen/bufgen.go

```diff
@@ -121,22 +121,24 @@ func GenerateWithBaseOutDirPath(baseOutDirPath string) GenerateOption {
 	}
 }
 
-// GenerateWithIncludeImports says to also generate imports.
+// GenerateWithIncludeImportsOverride says to always generate imports (or never
```

Comment:
Add a comment explicitly calling out that the given bool has presence, i.e. it's a pointer and this is a strict override.